### ISSUE TITLE
Update for sensor.sickchill fails

### ISSUE THREAD

**Issue by kobbra** [youdroid/home-assistant-sickchill#3]
sensor keeps failing with following error

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 279, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 472, in async_device_update
    await self.hass.async_add_executor_job(self.update)
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/sickchill/sensor.py", line 102, in update
    last_season = all_season_show['data'][str(nb_seasons)]
KeyError: '0'

Config looks like:
sensor:
  - platform: sickchill
    name: Wanted series
    host: !secret synology_ip
    token: !secret sc_token    
    port: 8899
    protocol: 'http'

**Comment by youdroid** [comment]
I will take a look at the problem, and I will let you know as soon as possible.

**Comment by youdroid** [comment]
I fixed it in [V1.0](https://github.com/youdroid/home-assistant-sickchill/releases/tag/V1.0) !

**Comment by kobbra** [comment]
Yes its working fine now! Thanks!

Regards,
Martin Johansson

Sent from my iPhone so excuse any typo's

7 juni 2020 kl. 22:37 skrev youdroid <notifications@github.com>:

﻿

I fixed it in V1.0<https://github.com/youdroid/home-assistant-sickchill/releases/tag/V1.0> !

**Comment by benjamfk** [comment]
I'm (still?) experiencing the same issue with Medusa (which is a fork, but the API should be the same).

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 278, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 471, in async_device_update
    await self.hass.async_add_executor_job(self.update)  # type: ignore
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/sickchill/sensor.py", line 117, in update
    last_season = all_season_show['data'][str(nb_seasons)]
KeyError: '81'
```
